### DISCUSSION TITLE
Report | Week 16

### DISCUSSION THREAD

**Discussion by LeonmAndreesen** [SE-TINF22B6/time2shine#87] [Show and tell]
Good day, 

in today's vlog we like to show you our chosen three metrics. First of all we will tell you something about them.

Web Application performance metrics:
 Web Application performance metrics are measurements used to evaluate and assess the efficiency, responsiveness, and overall health of a system. These metrics provide insights into how well an application is performing, both from a user perspective and in terms of technical performance. Some key metrics include:

User satisfaction and Apdex: Measures users’ satisfaction based on response time when interacting with your website or application.
FP/FCP (First Paint/First Contentful Paint): Indicates how quickly the first pixels are rendered after users access your website.
Time to interactive (TTI): Measures how long it takes for a page to become fully interactive for users.
Page speed and load time: Evaluates how quickly your web pages load.

Web application monitoring tool:
 A web application monitoring tool helps track and analyze the performance, availability, and security of web applications. It monitors metrics like response time, uptime, error rates, and resource utilization. By providing real-time insights, these tools help optimize application performance and enhance user experience

Software Testing Metrics:
Software metrics are quantitative measurements used to assess various aspects of software development, quality, and performance. These metrics help developers, managers, and stakeholders make informed decisions. Here are some common types of software metrics:

Size Metrics: Measure the size of software artifacts (e.g., lines of code, function points).
Complexity Metrics: Evaluate the intricacy of code (e.g., cyclomatic complexity, nesting depth).
Quality Metrics: Assess code quality (e.g., defect density, code coverage).
Effort Metrics: Estimate development effort (e.g., person-hours, story points)

These are the ones we chosen. Since we are a webgame these are the ones that make sense the most.
Software metrics are quantitative measurements used to assess various aspects of software development, quality, and performance. These metrics help developers, managers, and stakeholders make informed decisions. Here are some common types of software metrics:

![image](https://github.com/SE-TINF22B6/time2shine/assets/123726672/4f09019f-60ee-4220-84b1-8a3796895ccc)
![image](https://github.com/SE-TINF22B6/time2shine/assets/123726672/78726dce-294c-47c4-ab08-9c18a76e92c2)

To improve clarity in our clicker game, we made only clickable buttons "buyable," as shown in the following image:

![image](https://github.com/SE-TINF22B6/time2shine/assets/122756244/8167cfd7-a2ed-4ee4-ac40-0fa0021052fa)

**Comment by mirak55** [comment]
Nice work on the summary of all the different metrics, it is also reasonable why you chose those metrics. You have given a good overview over your metrics. Also keep up the good work on your website itself with the additions to the Röhrig Clicker.

**Comment by NicolasVien** [comment]
I appreciate your effort in summarizing the metrics and selecting the ones that are most relevant to your web games. It's important to have a clear understanding of the metrics that matter most to your webgame, and it seems like you're on the right track. Keep up the good work, !!!
